### DISCUSSION TITLE
Set custom Init Pattern

### DISCUSSION THREAD

**Discussion by Docmorzy** [bangcorrupt/hacktribe#223] [Q&A] [answered]
Hi,
Very enthusiastic noobs using Hacktribe !
Is it possible to setup the specs of the initialization pattern ?
When i initialize it's 87 bpm, scale mode on, Ragga 3, Kef F etc...
If I want my own canvas initializing like 100bpm, Scale mode off, C, Dorain or ever for example ?

**Comment by bangcorrupt** [comment, marked as answer]
If you use the latest patch the default pattern should be the same as the factory firmware.  See [set custom init pattern](https://github.com/bangcorrupt/hacktribe/wiki/How-To#set-custom-init-pattern) in the wiki to change it.

**Comment by Docmorzy** [comment]
Hello,
Is there any issue to rescue an OS who have not fully ended writing ?
Regards

**Reply by bangcorrupt** [reply]
I'm not sure I understand your question.  If the device won't boot see the [Debrick Guide](https://github.com/bangcorrupt/hacktribe/wiki/Debrick) in the wiki.  Also read these threads: #216, #79.  If you need help with that please start a new thread.
